_release_notes/armoryspinnaker_v2.18.1.md

```diff
@@ -0,0 +1,211 @@
+---
+layout: post
+title: v2.18.1 Armory Release (OSS Release 1.18.7)
+order: -21820200403172919
+hidden: false
+---
+
+# 04/03/20 Release Notes
+{:.no_toc}
+
+> Note: If you're experiencing production issues after upgrading Spinnaker, rollback to a [previous working version](http://docs.armory.io/admin-guides/troubleshooting/#i-upgraded-spinnaker-and-it-is-no-longer-responding-how-do-i-rollback) and please report issues to [http://go.armory.io/support](http://go.armory.io/support).
+
+* This is a placeholder for an unordered list that will be replaced with ToC. To exclude a header, add {:.no_toc} after it.
+{:toc}
+
+
+## Known Issues
+There are currently no known issues with this release.
+
+<!--- Example of a problem
+Igor added ..... which does.....
+
+**Symptoms:**
+**Fix:**
+-->
+
+
+
+## Highlighted Updates
+### Armory
+This release includes the following:
+  - Improved error handling in Pipelines as Code and its microservice, Dinghy.
+  - The Terraform integration now uses Clouddriver's built-in support for `git/repo` artifact types. Previously, you had to create a custom artifact that was a `git/repo` type.
+  - Improve edgecase handling in Policy Engine; support more complex policies that inclue other top-level declarations besides `deny`.
+  - Addresses a number of CVE's related to `springframework`, `tomcat`, and `jackson-databind` found in Spinnaker Java services
+
+
+
+###  Spinnaker Community Contributions
+REMOVE ME: link to the spninaker release: https://www.spinnaker.io/community/releases/versions
+REMOVE ME: maybe copy their notable changes here
+
```

Comment:
I'm not sure if there is anything to go here. Do you know @brian-armory ?

Reply:
Yeah, basically manually check the OSS version that correlates to this and copy/paste their highlights section. I just checked. None of the 1.18 edge releases have notable changes, so we can just add 

```
No changes to report.
```